doc/tree-menu.md

```diff
@@ -0,0 +1,43 @@
+# ASAB WebUI TreeMenu component
+
+This component is based on the library [react-simple-tree-menu](https://github.com/iannbing/react-simple-tree-menu).
+
+Displays menu items in a clear structure.
+
+## Example code
+
+```javascript
+import { TreeMenu } from 'asab-webui';
+...
+
+<TreeMenu
+	resource={resource}
```

Comment:
@aringocode If there is a resource in the example, there should be also `resources`, innit? :)